django_x509/tests/test_ca.py

```diff
@@ -394,3 +394,64 @@ def test_fill_subject_non_strings(self):
         x509 = crypto.X509()
         subject = ca2._fill_subject(x509.get_subject())
         self.assertEqual(subject.organizationName, 'Test CA')
+
+    def test_import_ca_validation_error(self):
+        certificate = """MIIEBTCCAu2gAwIBAgIBATANBgkqhkiG9w0BAQUFADBRMQswCQYDVQQGEwJJVDEL
```

Comment:
instead of redefining this, reuse the certificate available in `self.import_certificate`, but cut it in some way, you could do something like `self.import_certificate[20:]`